From 773cde9a496f1f2db6612927863d649bda2dc8da Mon Sep 17 00:00:00 2001
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Fri, 12 Mar 2010 22:13:43 +0800
Subject: [PATCH 02/11] Staging: remove unused #include <linux/version.h>

Remove unused #include <linux/version.h>('s) in
  drivers/staging/dt3155/allocator.c
  drivers/staging/rtl8192u/ieee80211/ieee80211_tx.c
  drivers/staging/vme/boards/vme_vmivme7805.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rtl8192u/ieee80211/ieee80211_tx.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

--- a/drivers/staging/rtl8192u/ieee80211/ieee80211_tx.c
+++ b/drivers/staging/rtl8192u/ieee80211/ieee80211_tx.c
@@ -47,7 +47,6 @@
 #include <linux/slab.h>
 #include <linux/tcp.h>
 #include <linux/types.h>
-#include <linux/version.h>
 #include <linux/wireless.h>
 #include <linux/etherdevice.h>
 #include <asm/uaccess.h>
